packages/components/pages/f-checkout/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout - Fozzie Checkout Component",
-  "version": "4.6.0",
+  "version": "4.6.1",
```

Comment:
Adding new props is definitely worth a minor bump, so `4.7.0`

Reply:
The props have been added in the past, but at that time the documentation was missing. I think it's fine to make it a patch bump. What do you think?

Reply:
Ah right, if you're just adding missing documentation then a patch is fine, carry on 😄